pydbtools/get_athena_query_response.py

```diff
@@ -39,7 +40,12 @@ def get_athena_query_response(
     necessary when using this in Python. Default is False.
     """
 
-    _, out_path = get_user_id_and_table_dir(force_ec2, region_name)
+    user_id, out_path = get_user_id_and_table_dir(force_ec2, region_name)
+
+    temp_db_name = get_database_name_from_userid(user_id)
+    sql_query = add_user_to_temp(sql_query, temp_db_name)
```

Comment:
This line will fail - as haven't imported it. 

I think it should be moved to utils and called `replace temp_database_name_reference` or some sort of gubbins like that?